FindPoetry.cmake

```diff
@@ -0,0 +1,35 @@
+#[=======================================================================[.rst:
+FindClang.cmake
+---------------
+
+This module is intended for use with ``find_package`` and should not be imported on
+its own.
+
+It will download and install the poetry package manager.
+#]=======================================================================]
+
+if(NOT WIN32)
```

Comment:
What's the motivation for the manual color output? CMake already does this to a certain extent, e.g. fatal errors are logged in red.

Reply:
Not much of one. Just made it easier to read.

Reply:
https://github.com/Opentrons/cmake-utils/pull/24/commits/e43541b5a4b4af141dd8c058979647aef35c7357